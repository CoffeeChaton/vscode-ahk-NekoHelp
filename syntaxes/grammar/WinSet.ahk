WinSet, AlwaysOnTop, OnOffToggle, WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Bottom, , WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Top, , WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Disable, , WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Enable, , WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Redraw, , WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Style, N, WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, ExStyle, N, WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Region, options, WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, Transparent, N, WinTitle, WinText, ExcludeTitle, ExcludeText
WinSet, TransColor, Color, WinTitle, WinText, ExcludeTitle, ExcludeText