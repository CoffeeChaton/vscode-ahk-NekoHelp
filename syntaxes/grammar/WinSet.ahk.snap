>WinSet, AlwaysOnTop, OnOffToggle, WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                    ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                 ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                           ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                    ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                                  ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Bottom, , WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#              ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#               ^ source.ahk meta.command.line.winset.ahk
#                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                 ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                           ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                    ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                  ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Top, , WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#           ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#            ^ source.ahk meta.command.line.winset.ahk
#             ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#              ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                       ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                        ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                 ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                              ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                               ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Disable, , WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#               ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                ^ source.ahk meta.command.line.winset.ahk
#                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                  ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                           ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                            ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                    ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                     ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                  ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                   ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Enable, , WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#              ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#               ^ source.ahk meta.command.line.winset.ahk
#                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                 ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                           ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                    ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                  ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Redraw, , WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#              ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#               ^ source.ahk meta.command.line.winset.ahk
#                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                 ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                           ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                    ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                  ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Style, N, WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#             ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#              ^^ source.ahk meta.command.line.winset.ahk
#                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                 ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                           ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                    ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                  ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, ExStyle, N, WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#               ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                ^^ source.ahk meta.command.line.winset.ahk
#                  ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                   ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                            ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                             ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                     ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                      ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                    ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Region, options, WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#              ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#               ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                       ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                        ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                 ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                  ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                           ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                        ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                         ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, Transparent, N, WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                    ^^ source.ahk meta.command.line.winset.ahk
#                      ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                       ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                 ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                         ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                          ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                       ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                        ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
>WinSet, TransColor, Color, WinTitle, WinText, ExcludeTitle, ExcludeText
#^^^^^^ source.ahk meta.command.line.winset.ahk support.function.command.winset.ahk
#      ^^ source.ahk meta.command.line.winset.ahk
#        ^^^^^^^^^^ source.ahk meta.command.line.winset.ahk keyword.command.option.param.winset.ahk
#                  ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                   ^^^^^^ source.ahk meta.command.line.winset.ahk
#                         ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                          ^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                   ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                    ^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                            ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                             ^^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk
#                                                          ^ source.ahk meta.command.line.winset.ahk punctuation.ahk
#                                                           ^^^^^^^^^^^^ source.ahk meta.command.line.winset.ahk